lib/compareValues.js

```diff
@@ -78,7 +80,9 @@ function compareValues (a, b) {
     case 'NULL':
       return spaceship(a, b)
     case 'PTYPE<TIME>':
-      return spaceship(a.getTime(), b.getTime())
+      const timeA = _.isDate(a) ? a.getTime() : a.epoch_time * 1000
```

Comment:
This wasn't exactly necessary - as I've avoided adding `timeFormat: 'raw'` to the PK-iterating queries, but I thought this shouldn't hurt.